alma-integrations/src/main/java/no/sikt/rsp/PartnerConverter.java

```diff
@@ -172,7 +172,7 @@ private ProfileDetails extractProfileDetails(Record record, final PartnerDetails
             partnerDetails.setAutoClaimSupported(AUTO_CLAIM_SUPPORTED);
             partnerDetails.setAutoClaimTime(AUTO_CLAIM_TIME);
         } else if (nncipUri.isPresent() && BaseBibliotekUtils.isNorwegian(record)) {
-            details.setProfileType(ProfileType.NCIP_P_2_P);
+            details.setProfileType(ProfileType.NCIP_P2P);
 
```

Comment:
Change in generated schema data caused this ti change and fail the build. Fixed now

Reply:
Why did it change? Do we generate code based on online, current schema, or a versioned schema in src/main/resources?

Reply:
I would suggest using current version that we download and check in in `alma-generated/src/main/resources` rather than using `https://developers.exlibrisgroup.com/wp-content/uploads/alma/xsd/rest_partners.xsd` in `alma-generated/build.gradle`

Reply:
But if we do that and the schema changes, things might start to fail without us knowing why. Is it not better to fail fast to discover the problem early? On the other hand, if we dont reach the schema uri when building the project because of some network error, it might also fail because of that. 

Awaiting your response before going further

Reply:
Up to you. But generally you can't reproduce any release at any given time if all build input is not in the repository.